lightning/backend/local.go

```diff
@@ -131,6 +141,53 @@ type local struct {
 	ingestConcurrency *worker.Pool
 	batchWriteKVPairs int
 	checkpointEnabled bool
+
+	tcpConcurrency int
+}
+
+// ConnPool is a lazy pool of gRPC channels.
+// When `Get` called, it lazily allocates new connection if connection not full.
+// If it's full, then it will return allocated channels round-robin.
+// This struct isn't goroutine-safe.
+type ConnPool struct {
```

Comment:
this should be package private? or I think we should make ConnPool concurrent safe